src/opencl_iwork_fmt_plug.c

```diff
@@ -0,0 +1,401 @@
+/* JtR format to crack iWork '09, and '13 / '14 files.
+ *
+ * This software is Copyright (c) 2015, Dhiru Kholia <kholia at kth.se> and
+ * Maxime Hulliger <hulliger at kth.se>, and it is hereby released to the
+ * general public under the following terms:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ *
+ * This code may be freely used and modified for any purpose.
+ *
+ * Big thanks to Sean Patrick O'Brien for making this format possible.
+ */
+
+#ifdef HAVE_OPENCL
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_opencl_iwork;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_opencl_iwork);
+#else
+
+#include <string.h>
+#include "aes.h"
+#include "hmac_sha.h"
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
```

Comment:
Do not mix system headers and local ones. First, include all `<system.h>` then we use to have a blank line, the you list all `"local.h"`